scan/lib/scan/options.rb

```diff
@@ -292,7 +297,19 @@ def self.available_options
                                      deprecated: "Use `--output_files` instead",
                                      conflicting_options: [:output_files],
                                      optional: true,
-                                     is_string: true)
+                                     is_string: true),
+        FastlaneCore::ConfigItem.new(key: :app_identifier,
+                                     env_name: 'SCAN_APP_IDENTIFIER',
+                                     optional: true,
+                                     description: "The bundle identifier of the app to uninstall (only needed when enabling reinstall_app)",
+                                     code_gen_sensitive: true,
+                                     default_value: ENV["SCAN_APP_IDENTITIFER"] || CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier),
+                                     default_value_dynamic: true),
+        FastlaneCore::ConfigItem.new(key: :reinstall_app,
```

Comment:
Instead of making `app_identifier` and `reinstall_app`, can we make this one option? Can we call it something like `reinstall_with_app_identifier`? This makes usage, doc, and code a bit cleaner to combine this into one (especially if `app_identifier` is only to be used with `reinstall_app`)

Reply:
I copied this from _snapshot_:

https://github.com/fastlane/fastlane/blob/665a23e6351702bbd6980fe1f7ec75a6f1f7bfa6/snapshot/lib/snapshot/options.rb#L119-L126

https://github.com/fastlane/fastlane/blob/665a23e6351702bbd6980fe1f7ec75a6f1f7bfa6/snapshot/lib/snapshot/options.rb#L104-L108

The question is if we want to live it like this to be consistent, make it simpler like you said, and/or make it like this for _snapshot_ too in the future which will get us hundreds of issues.

Reply:
@revolter Oh maaannnn, I didn't know _snapshot_ had it formatted like that 🤔 I don't think we can change _snapshot_ without it breaking some existing setups so I maybe we should keep this similar for consistency 😬